src/components/Homepage/Description.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
+
+interface IProps {
+  text: string;
+}
+
+export const Description: FC<IProps> = ({ text }) => (
+  <h1 className="p-1 0 inline m-0 font-gilbert text-4xl bg-white">
+    <strong className="relative">{text}</strong>
```

Comment:
why is here relative className ? and why is there strong use tailwind class for this